docs/pi-scripts/jackrun.bash

```diff
@@ -11,7 +11,7 @@ $WGET -O $WEBVER $NATION/$WEBVER
 if [ "$?" -ne "0" ]; then
   # not able to get the web version, so we will just continue on and run
   # This is likely the no network scenario or the server is down for maintenance
-  echo "could not get version from server"
+  echo "Cannot contact the nation:  no version from server"
```

Comment:
Change "from server" to "from $NATION" so that logs will be clear when trying with alternate hosts for testing or whatever